user/getting-started.md

```diff
@@ -31,9 +31,11 @@ drivers will be lost when you stop the container. To avoid this from happening,
 add the `-v` parameter to keep the `/var/lib/bblfshd` directory in a volume:
 
 ```bash
-$ docker run -d --name bblfshd --privileged -p 9432:9432 -v /tmp/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
+$ docker run -d --name bblfshd --privileged -p 9432:9432 -v /var/lib/bblfshd:/var/lib/bblfshd bblfsh/bblfshd
 ```
 
+(On macOS remove the parameter `-v /var/lib/bblfshd:/var/lib/bblfshd`.)
```

Comment:
I think a bit more information here would be nice, to better understand why this is needed and how it will affect.